srcpkgs/xmenu/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'xmenu'
+pkgname=xmenu
+version=4.5.4
+revision=1
+build_style=gnu-makefile
+makedepends="imlib2-devel libX11-devel libXft-devel libXinerama-devel"
+short_desc="X11 menu utility"
+maintainer="KawaiiAmber <japaneselearning101@gmail.com>"
+license="MIT"
+homepage="https://github.com/phillbush/xmenu"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=5c11dde7e5b07d1abd8fe338a264eec1a121a633f6b27e2bcef8baa4cebb82c9
+
+export LDFLAGS="-L${XBPS_CROSS_BASE}/usr/lib -lfontconfig -lXft -lX11 -lXinerama -lImlib2"
+CFLAGS="-I${XBPS_CROSS_BASE}/usr/include/freetype2"
```

Comment:
LDFLAGS doesn't need to be exported. But there's a possibly simpler way of fixing this without those variables.

`make_build_args="LOCALINC=$XBPS_CROSS_BASE/usr/include LOCALLIB=$XBPS_CROSS_BASE/usr/lib FREETYPEINC=$XBPS_CROSS_BASE/usr/include/freetype2 LDFLAGS+=\$(LIBS)"`

Reply:
Did vars in `make_build_args`

Reply:
I knew I was forgetting something `CFLAGS+=\$(INCS)`

Also, please break that line at ~80 columns.

Reply:
Added `CFLAGS+=\$(INCS)` and broke at ~80 columns.